plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -82,8 +82,10 @@ QSV_Encoder_Internal::QSV_Encoder_Internal(mfxVersion &version, bool isDGPU)
 	  m_nFirstSyncTask(0),
 	  m_outBitstream(),
 	  m_isDGPU(isDGPU),
-	  m_sessionData(NULL)
+	  m_sessionData(NULL),
+	  m_ver(version)
 {
+#if 0
```

Comment:
Is this intentional, or is this leftover from debugging?

Reply:
Thank you for pointing this out! This was intentional, as it is slow to create a VPL session. The only use for the session here was to query the version being used. I have moved the query version to another VPL session creation and also moved the logging texture vs sysmem to open.

Reply:
I think it looks fine to delete the code you commented out now, thanks.

Reply:
Looks like this has been removed now, so this thread can be resolved.